<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Halaman Keuangan</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>700</width>
     <height>500</height>
    </rect>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>300</y>
     <width>330</width>
     <height>150</height>
    </rect>
   </property>
   <property name="title">
    <string>Tambah Saldo</string>
   </property>
   <widget class="QLineEdit" name="input_tambah_saldo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string comment="&quot;00&quot;" extracomment="&quot;00&quot;"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Jumlah:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_tambah_saldo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Tambah Saldo</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>470</y>
     <width>330</width>
     <height>150</height>
    </rect>
   </property>
   <property name="title">
    <string>Tarik Saldo</string>
   </property>
   <widget class="QLineEdit" name="input_tarik_saldo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string comment="&quot;00&quot;" extracomment="&quot;00&quot;"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Jumlah:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_tarik_saldo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Tarik Saldo</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>110</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total Saldo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_saldo">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>150</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Total Saldo</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
